README.rst

```diff
@@ -37,6 +38,43 @@ for a complete list of features, flags, and documentation.
 jobs
 ^^^^
 
+create
+~~~~~~
+
+Create an data export job request and return the export job id.  There are two
+general workflows for creating data export requests.  To create a data export
+requests for all available schemas for a course::
+
+    courseraresearchexports jobs create schemas --courseId $COURSE_ID --statementOfPurpose "testing data export"
+
+If a more limited set of data is required, you can specify which schemas are
+included with the export.  (e.g. for the demographics tables)::
+
+    courseraresearchexports jobs create schemas --courseId $COURSE_ID --schemas demographics --statementOfPurpose "testing data export"
+
+If you are a data coordinator, you can request that user ids are linked between
+domains of the data export::
```

Comment:
Would it be possible to make this the default for data coordinators? I think almost never do people utilize the other form of export where schemas may not be linked with each other